Sources/SmokeDynamoDB/DynamoDBCompositePrimaryKeyTable.swift

```diff
@@ -68,6 +69,15 @@ public enum AttributeCondition {
     case beginsWith(String)
 }
 
+public enum WriteEntry<AttributesType: PrimaryKeyAttributes, ItemType: Codable> {
+    case update(new: TypedDatabaseItem<AttributesType, ItemType>, existing: TypedDatabaseItem<AttributesType, ItemType>)
+    case insert(new: TypedDatabaseItem<AttributesType, ItemType>)
+    case deleteAtKey(key: CompositePrimaryKey<AttributesType>)
```

Comment:
It's fine but I'm curious what the use case for these delete cases are (as opposed to just using the bulk delete api)?

Reply:
It is for the case where you have a mixture of statements you want to execute (insert a row, update another row, delete a third row). This would allow you to execute all those statements in a single batch request rather than making a seperate request for deletions.